src/purifier/noop.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// eslint-disable-next-line require-jsdoc
+export function dev() {
+  return {
+    assertElement: element => element,
```

Comment:
Consider using simple `console.assert` and `console.error` etc. instead of noops.

Reply:
I'm a bit torn on this: `dev()` asserts are removed in production anyway (if I understand correctly) so I'm unsure if the published library should implement them. I did implement `assertElement` and `user().error` now. `devAssert` and `userAssert` are unused (but required for rollup to figure out the dependency tree). Let me know your thoughts on this.